libbeat/outputs/pulsar/docs/pulsar.asciidoc

```diff
@@ -0,0 +1,75 @@
+[[pulsar-output]]
+=== Configure the Pulsar output
+
+++++
+<titleabbrev>Pulsar</titleabbrev>
+++++
+
+The Pulsar output sends events to Apache Pulsar.
```

Comment:
```suggestion

experimental[]

The Pulsar output sends events to Apache Pulsar.
```

Let's mark this as experimental initially.


Here's an example from the redis input: https://github.com/elastic/beats/blob/b9e20cb716cef2375b7e1077fe4113b8b528ae31/filebeat/docs/inputs/input-redis.asciidoc?plain=1#L10

It renders into the technical preview warning you can see at the top of https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-redis.html